src/main/java/uk/gov/hmcts/reform/logging/exception/AbstractLoggingException.java

```diff
@@ -5,6 +5,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+//Will be removed in future enhancement revision
+@Deprecated
```

Comment:
deprecation for easy inclusion into all the other modules before removing the exceptions.

Reply:
The story says to remove it...
happy for this to be split into multiple PRs...

Reply:
Yup that is what I'm intending.